### DISCUSSION TITLE
umi4 内如何配置 WPA Manifest

### DISCUSSION THREAD

**Discussion by caoguoli** [umijs/umi#12349] [Q&A]
umi版本 4.1.10 ，

在public 文件夹内建了 manifest.json：
{
  "name": "My UmiJS App",
  "short_name": "Umi App",
  "start_url": "/",
  "display": "standalone",
  "background_color": "#ffffff",
  "theme_color": "#2196f3",
  "icons": [
    {
      "src": "../src/favicon.jpg",
      "sizes": "192x192",
      "type": "image/png"
    }
  ]
}

src/document.ejs 内：<link rel="manifest" href="/manifest.json">，

build 后，无效

**Comment by fz6m** [comment]
umi 4 里是没有 `src/document.ejs` 这个东西的，如果要修改产物 html 内容，需要通过单独的 api ，参考 [这里](https://umijs.org/docs/introduce/faq#documentejs-%E5%8E%BB%E5%93%AA%E4%BA%86%E5%A6%82%E4%BD%95%E8%87%AA%E5%AE%9A%E4%B9%89-html-%E6%A8%A1%E6%9D%BF) 。

如还有问题，给一个最小复现看看吧。
